node/node.go

```diff
@@ -970,12 +970,41 @@ func (node *AlgorandFullNode) loadParticipationKeys() error {
 			}
 		} else {
 			// Tell the AccountManager about the Participation (dupes don't matter)
+			// make sure that all stateproof data (with are not the keys per round)
+			// are being store to the registry in that point
 			added := node.accountManager.AddParticipation(part)
 			if added {
 				node.log.Infof("Loaded participation keys from storage: %s %s", part.Address(), info.Name())
 			} else {
 				part.Close()
+				continue
 			}
+			// create a const value Number of records in one fetch
```

Comment:
this is a pretty long function. could we split it?

Reply:
Inner function that I could test separately maybe